test/generators/routes_generator_test.rb

```diff
@@ -8,7 +8,6 @@ class ControllerGeneratorTest < Rails::Generators::TestCase
   setup do
     prepare_destination
     provide_existing_routes_file
-    provide_existing_initializer_file
```

Comment:
this should have been removed when the routes flag was dropped from the config as its not needed in this test anymore